### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by samsmithnz** [aaubry/YamlDotNet#610] [General]
@aaubry, not sure if this is the right place, but I wanted to say thank you for continuing to maintain and enhance YamlDotNet. I've found it very useful in my side project, and I appreciate you continuing to drive it forward.  

thank you!

**Comment by aaubry** [comment]
Thank you for your kind words. I'm glad to know that this project is useful to you!

**Comment by gabriel-samfira** [comment]
I was actually hoping for this discussion to exist, because that's why I came here as well.

Thanks to all YamlDotNet maintainers for making this project what it is! Maintaining open source projects can at times be a thankless high stress endeavor, with little to no return other than helping out a community.

To any companies out there that are using this library (I know there are quite a few), please consider sponsoring the project.

**Comment by EdwardCooke** [comment]
Thanks @gabriel-samfira. I appreciate it. This is a fun project to maintain. It has a wide reaching audience and I do what I can to keep it moving along well.
